samples/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "sample",
+  "version": "0.0.1",
+  "description": "Snowflake Node.js driver samples",
+  "main": "jsonParserComparison.js",
+  "author": "",
+  "license": "ISC",
```

Comment:
should it be ISC or Apache as we have in main package.json? or we can omit it at all?